### PR TITLE
Add subcommand config

### PR DESCRIPTION
**Pull request by kengo0924**
**What this PR does / why we need it**:

ika no bunsyou ha funiki to google-honyaku ni yotte sakusei simasita.

### request to
It is troublesome to specify config every time when switching some datadog organizations, so add sub-command `config`.
- dogleash config list/get/set
- location of config: `~/.dogrc.d/`
  - `~/dogrc.d/hoge`
  - `~/dogrc.d/fuga`
  - ...

#### working now
  - vipering
  - refactor
  - handling errors

#### how to use
```bash
$ doglesh config list
fuga
hoge
```

```bash
$ dogleash config get
hoge
```

```bash
$ dogleash config set fuga
```

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
nothing

Fixes #[PUT ISSUE NUMBER HERE]
nothing

**Special notes for your reviewer** *(optional)*:
yoroshiku onegai simasu

### CODE REVIEW

**Comment by kengo0924** [comment]
## update

* config path
~/.dogrc (default dog
~/.config/dogleash/config.yml (default dogleash

### config sample
```
current: dogrc
organizations:
- keys:
    apikey: hoge
    appkey: hogehoge
  name: hoge
- keys:
    apikey: fuga
    appkey: fugafuga
  name: fuga
```

File: cmd/config/root.go [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+// orgs
+type Organization struct {
+	Name   string `mapstructure:"name"`
+	APIKey string `mapstructure:"apikey"`
+	APPKey string `mapstructure:"appkey"`
+}
+
+// DC dogleash config
+var DC DogleashConfig
```
Comment (toVersus) [inline cmd/config/root.go]:
```suggestion
var DC cmd.DogleashConfig
```

Then, you can delete duplicated declarations of both DogleashConfig and Organization above.

File: cmd/config/root.go [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+// DC dogleash config
+var DC cmd.DogleashConfig
+
+// config path
+var dogrcFile = filepath.Join(os.Getenv("HOME"), ".dogrc")
+var dogleashFile = filepath.Join(os.Getenv("HOME"), ".config/dogleash/config.yml")
+
+var configCmd = &cobra.Command{
+	Use:   "config",
+	Short: "Perform operations related to config of Datadog API/APP keys",
```
Comment (toVersus) [inline cmd/config/root.go]:
```suggestion
	Short: "Manage config files of Datadog API/APP keys",
```

File: cmd/config/set.go [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+	"os"
+	"strconv"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var configSetCmd = &cobra.Command{
+	Use:   "set",
+	Short: "Set a Config for API/APP keys",
```
Comment (toVersus) [inline cmd/config/set.go]:
```suggestion
	Short: "Set API/APP keys per organization",
```

**Comment by tani-yu** [comment]
Thank you PR.
If you will change to use configuration file for dog leash. you should explain in README.

**Pull request merged**